File: C#软件Dll加密混淆工具/SDK/IntelliLock Example Projects/Default/Licensed Control Library 2/VB.NET/VS 2012/Licensed Control/InstallLicenseForm.vb
Public Class InstallLicenseForm

    Private Sub okbutton_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles okbutton.Click
        Close()
    End Sub

    ''' <summary>
    ''' Shows an open file dialog and installs the selected license file
    ''' </summary>   
    Private Sub installbutton_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles installbutton.Click
        If (Me.openFileDialog1.ShowDialog = DialogResult.OK) Then

            ' Embedded the selected license file within the corresponding main application.
            IntelliLock.Licensing.EvaluationMonitor.StoreLicenseAsResource(Me.openFileDialog1.FileName)

            ' Load the license file to update the current license status. The next time you run the main application the stored
            ' license file will be found and loaded automatically.
            IntelliLock.Licensing.EvaluationMonitor.LoadLicense(Me.openFileDialog1.FileName)
        End If
    End Sub
End Class

File: C#软件Dll加密混淆工具/SDK/IntelliLock Example Projects/Default/Licensed Control Library 2/C#/VS 2005/Licensed Control/InstallLicenseForm.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using IntelliLock.Licensing;

namespace Licensed_Control
{
    public partial class InstallLicenseForm : Form
    {
        public InstallLicenseForm()
        {
            InitializeComponent();
        }

        /// <summary>
        /// Shows an open file dialog and installs the selected license file
        /// </summary>   
        private void installbutton_Click(object sender, EventArgs e)
        {
            if (openFileDialog1.ShowDialog() == DialogResult.OK)
            {
                // Embedded the selected license file within the corresponding main application.
                //
                EvaluationMonitor.StoreLicenseAsResource(openFileDialog1.FileName);

                // Load the license file to update the current license status. The next time you run the main application the stored
                // license file will be found and loaded automatically.
                //
                EvaluationMonitor.LoadLicense(openFileDialog1.FileName);
            }
        }

        private void okbutton_Click(object sender, EventArgs e)
        {
            this.Close();
        }
    }
}